aten/src/ATen/native/native_functions.yaml

```diff
@@ -12273,6 +12273,21 @@
   dispatch:
     CompositeExplicitAutograd: _test_warn_in_autograd
 
+# Note: this function is only for testing.
+- func: _test_autograd_multiple_dispatch.one(Tensor self) -> Tensor
```

Comment:
What if the overload suffixes were more descriptive of what's going on, rather than just "one" and "two"? Like, IIUC from `derivatives.yaml`, `.one` is a "full coverage" test whereas `.two` is NT only. I think maybe using something like those (or whatever the correct characterizations are) as suffixes would help readability, be a good connective thread between the different moving parts here, etc